### PR TITLE
Fixes #161: Fixed the LokLak Web Tweets app

### LINKED ISSUE #161: LokLak Web Tweets app is not working
- Issue type: Bug report

### Short description
LokLak Web Tweets app is not working as there are changes in query syntax in the server side.

### Screenshots of the issue
screenshot 122

### Would you like to work on the issue?
Yes. I will work on this issue.

### PR DESCRIPTION
**Pull request by kavithaenair**
### Short description
Fixes #161 
Demo link : https://kavithaenair.github.io/webtweets/

**Screenshots for the change:**

screenshot 124

I have:
- [x] There is a corresponding issue for this pull request.
- [x] Mentioned the Issue number in the pull request commit message `Fixes #<number> commit message`
- [x] There is only strictly only one commit per issue.

### For the reviewers
I have:
- [ ] Reviewed this pull request by an authorized contributor.
- [ ] The reviewer is assigned to the pull request.

### CODE REVIEW

**Comment by kavithaenair** [comment]
@djmgit @SKrPl @daminisatya @mariobehling open for review.

**Comment by djmgit** [comment]
@kavithaenair please check the travis failures

**Comment by kavithaenair** [comment]
@djmgit 
The build is failing because of the image link. If am following the way written for coala bears, the link is taking white spaces:
https://camo.githubusercontent.com/a6677b08c955af8400f44c6298f40e7d19cc5b2d/68747%20470733a2f2f73332e616d617a6f6e6177732e636f6d2f6769746875622f726962626f6e732f666f72%206b6d655f72696768745f677261795f3664366436642e706e67

This is the correct link: 
https://camo.githubusercontent.com/a6677b08c955af8400f44c6298f40e7d19cc5b2d/68747470733a2f2f73332e616d617a6f6e6177732e636f6d2f6769746875622f726962626f6e732f666f726b6d655f72696768745f677261795f3664366436642e706e67

How can i solve this?

**Comment by djmgit** [comment]
@kavithaenair so the problem is with length of the image url. You may download the image and use it.

File: webtweets/index.html [hunk 1]
```diff
@@ -65,13 +65,7 @@ <h1 class="large-h1 text-white">Twitter Updates</h1>
...
             </section>
         </div>
-        <a href="https://github.com/fossasia/fossasia-loklak-webtweets">
-            <img style="z-index:1;position: absolute; top: 0; right: 0; border: 0;"
-                 src="https://camo.githubusercontent.com/a6677b08c955af8400f44c6298f40e7d19cc5b2d/68747
-                   470733a2f2f73332e616d617a6f6e6177732e636f6d2f6769746875622f726962626f6e732f666f72
-                   6b6d655f72696768745f677261795f3664366436642e706e67"
-                 alt="Fork me on GitHub"
-                 data-canonical-src="https://s3.amazonaws.com/github/ribbons/forkme_right_gray_6d6d6d.png">
+        <a href="https://github.com/fossasia/fossasia-loklak-webtweets"><img style="z-index:1;position: absolute; top: 0; right: 0; border: 0;" src="https://camo.githubusercontent.com/a6677b08c955af8400f44c6298f40e7d19cc5b2d/68747470733a2f2f73332e616d617a6f6e6177732e636f6d2f6769746875622f726962626f6e732f666f726b6d655f72696768745f677261795f3664366436642e706e67" alt="Fork me on GitHub" data-canonical-src="https://s3.amazonaws.com/github/ribbons/forkme_right_gray_6d6d6d.png">
```
Comment (djmgit) [inline webtweets/index.html]: Instead of using this long link you can save the image and provide the path

Comment (kavithaenair) [inline webtweets/index.html]: Okay thank you. Will do that.

**Comment by kavithaenair** [comment]
@djmgit build is successful now. Thank you for the suggestion. 😄

**Comment by djmgit** [comment]
@kavithaenair I tried using the test link you have provided above, however its only showing fetching tweets. The console is showing jquery reference error.

**Comment by kavithaenair** [comment]
@djmgit can you be more specific about the error message. For me it shows the scripts to be loaded are not so safe but i gave permissions then the tweets were visible to me.

**Comment by djmgit** [comment]
@kavithaenair 

webtweets

**Comment by kavithaenair** [comment]
@djmgit the part which you are mentioning about was already present in code. I have not changed that part of the code. As i mentioned before, it is showing errors in the console when the scripts (as they are unsafe) are not loaded:
image
 But when i allow the scripts, its showing warning:
image

**Pull request merged**